jenkins/run

```diff
@@ -27,13 +34,9 @@ fi
 
 _branch_name="$(cat jenkins/branch.name)"
 
-if [[ -z "${IMAGE_REPO}" ]]; then
-    IMAGE_REPO="quay.io/pbench"
-fi
-
-if [[ -z "${IMAGE_KIND}" ]]; then
-    IMAGE_KIND="fedora"
-fi
+_image_repo=${IMAGE_REPO:-quay.io/pbench}
+_image_role=${IMAGE_ROLE:-devel}
+_image_kind=${IMAGE_KIND:-fedora}
```

Comment:
I wish we could back away from this practice of putting leading underscores on everything.  Yes, there are situations in which doing so is warranted, but I don't think that shell variables at the top level of a script are one of them, and thus they are just  unproductive visual clutter.

Reply:
The typical use I have encountered is to separate internal from external.  Does this pose a serious support problem?

Reply:
In this context, there is no "external" (or, for that matter, "internal").

The underscore adds a useless and unneeded character to each name; when several of these names appear together on the line, it uselessly extends the line.  Underscores are normally used on the interior of symbols, so adding them on the exterior makes the symbol harder to parse, visually.  Shell scripts already suppress most use of horizontal space, so adding extra underscores compounds the problem.

In short, they make the code harder to read.

For cases where we want to make it explicit that a variable has limited scope (e.g., if it is declared `local` inside a defined shell function), the cost is potentially worthwhile.  But, these variables have normal scope, so there is nothing to be gained by trying to claim that they are "local".  And, they lead to this (which doesn't fit on an 80-column line):
```
_image=${IMAGE:-${_image_repo}/pbench-${_image_role}-${_image_kind}:${_branch_name}}
```
which should look more like this:
```
image=${IMAGE:-${image_repo}/pbench-${image_role}-${image_kind}:${branch_name}}
```